README.md

```diff
@@ -10,6 +9,52 @@
   </a>
 </div>
 
+<br>
+<div align="center">
+    <img 
+        src="https://img.shields.io/github/license/Avaiga/taipy?style=plastic&color=ff371a&labelColor=1f1f1f" 
+        alt="GitHub License" 
+        height="20px" 
+    />
+    &nbsp;
+    <img 
+        src="https://img.shields.io/badge/python%20version-3.8-ff371a?style=plastic&labelColor=1f1f1f" 
```

Comment:
This is incorrect. We don't support 3.8 anymore.

Please add also versions 3.9, 3.10, 3.11, 3.12

Thx

Reply:
I have added 3.8 based on [this](https://github.com/Avaiga/taipy/issues/1934#issuecomment-2397818143) comment from @RymMichaut .
Sure I will add badges for the other version and will remove 3.8.